modules/nw-ne-comparing-ingress-route.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * ingress/understanding-networking.adoc
+
+[id="nw-ne-comparing-ingress-route_{context}"]
+= Comparing Routes and ingresses
```

Comment:
In addition to what @ctauchen discusses, just wondering if you're trying to compare the actual objects or not. Because if so, then you could do Comparing Route and Ingress objects. (Headings are the only place we don't use backticks for object names.

And though some if this is definitely because I don't know much about ingress/routes, the explanation in this section isn't very clear to me what the differences are. I think it would be helpful to start out saying that one is a Kubernetes object and one is an openshift one. And then try to explain when you might want to use one over the other (if there are feature differences).

This is probably more detailed than you want or need here, but you could look at this for inspiration: https://deploy-preview-32226--osdocs.netlify.app/openshift-enterprise/latest/applications/deployments/what-deployments-are.html#deployments-comparing-deploymentconfigs_what-deployments-are

That compares Deployment and DeploymentConfig objects. But it right away tells you which is which, and what they share in common, and then where they differ.